### DISCUSSION TITLE
Is it possible to force Renovate to use specific poetry version?

### DISCUSSION THREAD

**Discussion by jacek-jablonski** [renovatebot/renovate#22596] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

Now Renovate uses poetry version 1.4.2. Is it possible to force renovate to use poetry 1.5.1?

### What you tried so far.

Reading the docs.

**Comment by rarkins** [comment, marked as answer]
Try configuring constraints.poetry in repo config
